### ISSUE TITLE
events model + html rendering

### ISSUE THREAD

**Issue by gregsadetsky** [gregsadetsky/nycnoise#44]
this is what a few sample events look like on nyc-noise.com:

<img width="630" alt="image" src="https://github.com/gregsadetsky/nycnoise/assets/1017304/6f5137e4-3f7b-4f10-a92d-d6596fb122b8">

---

there are many parts to an event listing :-) and we want to recreate the main parts for now, both in the model (i.e. the database) and in the html template/rendering.

task: create the relevant fields in the model, as well as a sub-template that will be included from the homepage and other pages (see #36) to render a single event.

---

an event model should include the following:
- (we are skipping the accessibility icons for now -- we'll discuss/see how to go about them once venues are imported)
- optional "bracket" content i.e. `[OUTDOORS!]` that sometimes appears at the very beginning.
- datetime
  - it would be great if the admin defaulted to new york time. we do however (definitely) want to keep the underlying database/project datetime to be utc.
- presenter/title - this is the part that comes after the hour and is not bold
- artists (this will link to the main hyperlink -- see below)
- main hyperlink to the concert/tickets - optional (either render an `<a>` or don't if there's no link)
- potentially a second link name + url
- "@" character followed by the venue name + address. we can start with the venue name only for now, and as our models become more complicated / we import the venues (see #15) we'll add more stuff here
- optional "tiny extra text"
- tiny extra text
- optional entire override -- this should be a field (which should rich text -- see #10) which allows Jessica to entirely override the whole event text. this is the escape hatch :-)

**Comment by SwantjeM** [comment]
instead of start and end times, have a duration instead?

**Comment by evanconway** [comment]
It might be best to leave start and end times in the model, and infer duration from those. The Calendar formats I've seen so far require a start and end time.

**Comment by gregsadetsky** [comment]
yeah @SwantjeM @evanconway I think both can make sense - duration or end time.

but yeah - to me, end time does feel a little bit more "natural" in django/python in the sense that an end time is "just" a native python datetime object, and the django admin can show a date+time input for that. a duration would be just... a number? of minutes?

speaking of, I didn't know that django had "[DurationField](https://docs.djangoproject.com/en/4.2/ref/models/fields/#durationfield)"

in any case, my vote is for end time :-)

**Comment by gregsadetsky** [comment]
after talking with @SwantjeM and discussing both the current state of dev, next steps, what we want (or don't want) to add, putting this issue on hold for now. we'll get back to it, somehow!
